wordpress-mobile/WordPress-Android
# Update Gutenberg to 0.3.2

This includes a fix to capitalize text by default in Gutenberg

To test:
- Create a new Gutenberg post and make sure the first letter in the block and after every period is capitalized in paragraphs.

Update release notes:

- [x] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`. Internal only
